File: src/cn/langya/sun/ui/impl/notification/NotificationType.java
package cn.langya.sun.ui.impl.notification;

import java.awt.*;

public enum NotificationType {
    SUCCESS(new Color(0x60E092)),
    ERROR(new Color(0xFF2F2F)),
    WARNING(new Color(0xF5FD00)),
    INFO(new Color(0x6490A7));

    private final Color renderColor;

    NotificationType(Color renderColor) {
        this.renderColor = renderColor;
    }

    public Color getRenderColor() {
        return renderColor;
    }
}

File: src/cn/langya/sun/ui/impl/notification/NotificationType.kt
package cn.langya.sun.ui.impl.notification

import java.awt.*;

enum class NotificationType(var renderColor:Color) {
    SUCCESS(Color(0x60E092)),
    ERROR(Color(0xFF2F2F)),
    WARNING(Color(0xF5FD00)),
    INFO(Color(0x6490A7));
}
